front/cypress/e2e/project_page_timeline.cy.ts

```diff
@@ -74,7 +74,7 @@ describe('New timeline project', () => {
       title: projectTitle,
       descriptionPreview: projectDescriptionPreview,
       description: projectDescription,
-      publicationStatus: 'published',
+      publicationStatus: 'draft',
```

Comment:
Does this mean we only have a test for draft projects now and not published? Probably not a big issue, but published is probably the more likely scenario.

Reply:
Double checked - we do have an existing one actually :) So we're all good :+1: